data/json/items/ammo_types.json

```diff
@@ -430,6 +430,12 @@
         "id" : "unfinished_char",
         "name" : "semi-charred fuel",
         "default" : "unfinished_charcoal"
+    },
+    {
+        "type" : "ammunition_type",
+        "id" : "gas_fungicidal",
+        "name" : "fungicidal gas",
+        "default" : "gas_fungicidal"
```

Comment:
Sounds like you only changed it in ammo_types, but not in ammo.json: "gas_fungicidal" had the old "ammo_type" field.

"ammo" field refers to the ammo type id. So if you're changing ammo id to "chemical_spray", the fungicidal gas needs `"ammo_type": "chemical_spray"`.